blockchain/checkpoints.go

```diff
@@ -259,3 +260,31 @@ func (b *BlockChain) IsCheckpointCandidate(block *btcutil.Block) (bool, error) {
 	// All of the checks passed, so the block is a candidate.
 	return true, nil
 }
+
+// maybeUpdateMostRecentCheckpoint potentially updates the most recently known
+// checkpoint to the provided block node.
+//
+// This function MUST be called with the chain lock held (for writes).
+func (b *BlockChain) maybeUpdateMostRecentCheckpoint(node *blockNode) {
```

Comment:
This introduces a hard fork. We don't want this function for our code.

Reply:
Removed